remoteappmanager/docker/container_manager.py

```diff
@@ -165,7 +158,7 @@ def _start_container(self, user_name, image_name, volumes):
         # build the dictionary of keyword arguments for create_container
         container_name = _generate_container_name("remoteexec",
                                                   user_name,
-                                                  image_name)
+                                                  mapping_id)
```

Comment:
Then we have two completely different levels of accounting.

1. the number of different applications an individual user can start.
2. the number of containers a team of users can spawn of the same application.

These are different. My point is that user A should be allowed to run one container of application B with configuration C. He can however run a second container of application B with configuration C2, or a second container of application B2 with configuration C. Each of these containers will have different mapping_id.